content/docs/2.1/operate/cluster.md

```diff
@@ -36,12 +36,12 @@ Here is an overview of all KEDA deployments and the supported replicas:
 
 | Deployment     | Support Replicas        | Reasoning                     |
 |----------------|-------------------------|-------------------------------|
-| Operator       | 1                       |                               |
+| Operator       | 2                | While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic. You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce a downtime during a failover.|
```

Comment:
here's some possible alternative wording. I won't be offended if you want to ignore this 😄 

```suggestion
| Operator       | 2                | While you can run more replicas of our operator, only one operator instance will be active. The rest will be standing by, which may reduce downtime during a failure. Multiple replicas will not improve the performance of KEDA. |
```